File: app/src/main/java/com/examle/curexchange/data/remote/ApiCryptoCode.java
package com.examle.curexchange.data.remote;


import com.examle.curexchange.data.model.pojo.CryptoCode;

import io.reactivex.Flowable;
import retrofit2.http.GET;

public interface ApiCryptoCode {

    @GET("currencies")
    Flowable<CryptoCode> getCryptoCodes();
}

File: app/src/main/java/com/examle/curexchange/data/remote/ApiCryptoCode.kt
package com.examle.curexchange.data.remote

import com.examle.curexchange.data.model.pojo.CryptoCode
import io.reactivex.Flowable
import retrofit2.http.GET


interface ApiCryptoCode {

    @GET("currencies")
    fun getCryptoCodes(): Flowable<CryptoCode>
}